usaspending_api/awards/models.py

```diff
@@ -455,7 +455,7 @@ class TransactionAssistance(DataSourceTrackedModel):
     uri = models.TextField(blank=True, null=True)
     cfda_number = models.TextField(blank=True, null=True, verbose_name="CFDA Number")
```

Comment:
To close the loop publicly, front end is using these old fields, and we hadn't planned for their time when estimating the story, so we'll keep these redundant fields for now. Hopefully when the front-end has to use some of the new data we just surfaced, we'll be able to do this bit of cleanup.